clusters/build-clusters/02_cluster/assets/admin_cluster-admin-0_list.yaml

```diff
@@ -0,0 +1,27 @@
+kind: List
+apiVersion: v1
+items:
+- kind: Group
+  apiVersion: v1
+  metadata:
+    name: ci-admins
+  users:
+  - AlexNPavel
+  - bbguimaraes
+  - droslean
+  - hongkailiu
+  - petr-muller
+  - stevekuznetsov  
+
+- apiVersion: rbac.authorization.k8s.io/v1
+  kind: RoleBinding
+  metadata:
+    name: admin
+    namespace: ci
```

Comment:
@cgwalters 
We prefer `ns` ci if it is not occupied.
Otherwise, need a `ns` here.